ci/kubetest/test_clickhouse_custom_settings_profile.py

```diff
@@ -0,0 +1,60 @@
+import json
+
+import pytest
+
+from utils import cleanup_k8s, install_chart, kubectl_exec, wait_for_pods_to_be_ready
+
+VALUES_YAML = """
+cloud: local
+
+clickhouse:
+  profiles:
+    default/max_execution_time: 90
+    default/max_memory_usage: 20000000000
+
+#
+# For the purpose of this test, let's disable everything else
+#
+
+# PostHog
+web:
+  enabled: false
+migrate:
+  enabled: false
+events:
+  enabled: false
+worker:
+  enabled: false
+plugins:
+  enabled: false
+
+# Services
+postgresql:
+  enabled: false
+redis:
+  enabled: false
+kafka:
+  enabled: false
+ingress:
+  enabled: false
+pgbouncer:
+  enabled: false
+"""
+
+
+@pytest.fixture
+def setup(kube):
+    cleanup_k8s()
+    install_chart(VALUES_YAML)
+    wait_for_pods_to_be_ready(kube)
+
+
+def test_custom_clickhouse_settings_profile(setup, kube):
+    command = f"clickhouse-client -q \"SELECT name, value FROM system.settings WHERE name IN ('max_execution_time', 'max_memory_usage') ORDER BY name FORMAT JSON\""
+    result = kubectl_exec("chi-posthog-posthog-0-0-0", command)
```

Comment:
Nit: not sure about it but `kubetest` might expose a method on the container object for this already

Reply:
https://kubetest.readthedocs.io/en/latest/reference.html#pod Doesn't seem like it. However I could probably use the HTTP API for this + http_proxy methods. Will experiment!

Reply:
This almost works - ran into a hitch where authentication always fails with this clickhouse-side error:

```
2022.01.25 15:09:23.030568 [ 60 ] {} <Warning> AddressPatterns: Failed to check if the allowed client hosts contain address ::ffff:10.244.0.1. DB::Exception: Cannot getnameinfo(::ffff:10.244.0.1): Name or service not known, code = 198
```

Will need to dig deeper tomorrow.

Reply:
Dug a bit more - I don't think it's worth investing more here, seems like a deeper clickhouse-operator related bug than worth tackling here.